massa-consensus-worker/src/worker/mod.rs

```diff
@@ -35,6 +36,18 @@ pub struct ConsensusWorker {
     next_instant: Instant,
 }
 
+#[derive(Clone)]
+pub struct WsConfig {
+    /// Whether WebSockets are enabled
+    pub enabled: bool,
+    /// Broadcast sender(channel) for blocks headers
+    pub block_header_sender: Sender<BlockHeader>,
+    /// Broadcast sender(channel) for blocks
+    pub block_sender: Sender<Block>,
+    /// Broadcast sender(channel) for filled blocks
+    pub filled_block_sender: Sender<FilledBlock>,
+}
```

Comment:
Was there any discussion I'm unaware of where it was decided that we'd use tokio channels for this? Why not use crossbeam instead?

Reply:
IMHO the new library is built on top of Tokio so it seems to be a safe choice.

this is the [example](https://github.com/paritytech/jsonrpsee/blob/master/examples/examples/ws_pubsub_broadcast.rs) which I followed to dev this feature.

https://github.com/massalabs/massa/issues/3076